presto-testing/pom.xml

```diff
@@ -92,6 +92,16 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Maybe you want `presto-testing-kafka`?

Reply:
Do you mean I should create a separate module?

Reply:
Yes